scss/functions/_utilities-map.scss

```diff
@@ -0,0 +1,89 @@
+// Builds a map of utility classes.
+@function build-utilities-map($grid-breakpoints: $grid-breakpoints, $utilities: $utilities) {
+  // Build a breakpoint map that does not include the zero breakpoint.
+  $breakpoints-map: ();
+  @each $name, $min in $grid-breakpoints {
+    @if $min != 0 {
+      $breakpoints-map: map-merge(
+        $breakpoints-map,
+        (
+          "-" + $name: $name,
+        )
+      );
+    }
+  }
+
+  $utilities-map: () !default;
+
+  @each $key, $utility in $utilities {
+    @if type-of($utility) == "map" {
```

Comment:
[type-of($value) //=> unquoted string](https://sass-lang.com/documentation/modules/meta#type-of)

Reply:
We're inconsistent in this across our source code, so going to leave as-is for now.